src/servicebus/azext_servicebus/_utils.py

```diff
@@ -0,0 +1,35 @@
+from azure.mgmt.servicebus.models.service_bus_management_client_enums import SkuName, SkuTier, AccessRights
```

Comment:
The SDK is not under azure.mgmt.servicebus since it is now included in the extension.
I think this should be `from azext_servicebus.servicebus...`